aeppl/joint_logprob.py

```diff
@@ -47,11 +46,11 @@ def factorized_joint_logprob(
         Y \sim& \operatorname{N}(0, \sigma^2)
 
     If we create a value variable for ``Y_rv``, i.e. ``y = at.scalar("y")``,
```

Comment:
```suggestion
    If we create a value variable for ``Y_rv``, i.e. ``y_vv = at.scalar("y")``,
```

This only adds the `*_vv` suffix convention for value variables.